go/services/cluster_metadata/internal/handlers/metadata_handler.go

```diff
@@ -15,10 +14,7 @@ import (
 
 func NewMetadataRouter(metadataHandler *MetadataHandler, rootRouter *mux.Router) *MetadataRouter {
 	router := rootRouter.PathPrefix("/metadata").Subrouter()
-	router.Methods(http.MethodGet).Path("/clusters/{clusterId}/applications").HandlerFunc(metadataHandler.GetClusterMetadataForTimerange)
-	router.Methods(http.MethodGet).Path("/clusters/{clusterId}/nodes").HandlerFunc(metadataHandler.GetNodeMetadataForTimerange)
-	router.Methods(http.MethodGet).Path("/clusters").HandlerFunc(metadataHandler.GetClusterList)
-	router.Methods(http.MethodPost).Path("/clusters").HandlerFunc(metadataHandler.InsertClusterMetadata)
```

Comment:
Just to make sure. The event driven architecture in management-service is will allow collecting the cluster metadata starting with this PR, right?

Reply:
Correct